src/dynarmic/backend/x64/constant_pool.h

```diff
@@ -29,7 +30,13 @@ class ConstantPool final {
 private:
     static constexpr size_t align_size = 16;  // bytes
 
-    std::map<std::tuple<u64, u64>, void*> constant_info;
+    struct ConstantHash {
+        std::size_t operator()(const std::tuple<u64, u64>& constant) const {
```

Comment:
```suggestion
        std::size_t operator()(const std::tuple<u64, u64>& constant) const noexcept {
```

since hashing functions should never throw